### ISSUE TITLE
fenced-code-blocks does not capture the optional language correctly

### ISSUE THREAD

**Issue by ngkan** [trentm/python-markdown2#396]
`_fenced_code_block_re` looks for the first non-empty line to capture the optional language; it should only look at the first line (the line with triple backticks).
```python
>>> import markdown2
>>> markdown2.markdown("""
... ```
... a
... b
... c
... ```
... """, extras=['fenced-code-blocks'])
'<pre><code>b\nc\n</code></pre>\n'
>>> 
```

**Comment by dantezhu** [comment]
me too!

**Comment by nicholasserra** [comment]
Fixed in #408
